pymeasure/instruments/rohdeschwarz/ngpx.py

```
@@ -0,0 +1,569 @@
+
+        When ``True``, all tracking-selected channels operate in tracking
+        mode (voltage setpoints follow the primary/master channel).
+        Returns ``False`` if no channels are tracking-selected.
+        """
+        ch_list = self._get_tracking_selected_list()
+        if not ch_list:
+            return False
+        try:
+            response = self.ask(f"TRAC?{ch_list}")
+            return bool(int(response.strip()))
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)